src/components/ToolBox.vue

```diff
@@ -37,18 +46,27 @@ export default class ToolBox extends Vue {
     return this.tool.Amount > 1000 ? this.tool.Amount : "";
   }
 
-  private clickBuyButton = () => {
+  private clickBuyButton() {
```

Comment:
why not arrow function?

Reply:
i was getting some odd issues with scoping, `this.tool` was coming up null/undefined.